orchestrator-salt/src/main/resources/salt/salt/cloudera/agent/init.sls

```diff
@@ -8,6 +8,8 @@ install-cloudera-manager-agent:
     - pkgs:
       - cloudera-manager-daemons
       - cloudera-manager-agent
+    - unless:
+      - rpm -q cloudera-manager-daemons cloudera-manager-agent
```

Comment:
I am not sure whether it matters but it breaks ubuntu and debian

Reply:
The unless execution will fail on ubuntu (due to lack of rpm command), which is going to make the state execute.